### DISCUSSION TITLE
I would like to ask how to use Etsy API to export new order data？

### DISCUSSION THREAD

**Discussion by mycx001** [etsy/open-api#1336] [Q&A]
My friend and I both have shops, and we sell customized products. Previously, we had to export new orders from the shop backend and then find note and gift message on the new order interface. This was too cumbersome and time-consuming. I have heard about the API, which seems to be very convenient and safe, but I don't know how to program, so who can help me? Or tell me where to learn relevant knowledge, I can't understand the official documentation.

Looking forward to your reply.

**Comment by turboturtleD** [comment]
Coding is a pretty deep rabbit hole.  Unless someone has a commercially approved app that suits your need, I don’t think we can (or are allowed to) access shops other than the one for which the API key was generated for.  If you can manage to obtain an API key for your shop, someone might be able to help you in the shadows - your request is pretty straightforward and simple.  If you want to obtain an API key, I recommend searching through the GitHub Q & A for things to say and things not to say in your API application.  If you tell Etsy you plan to give your key to someone else, they will not grant you access to the key/API.

**Reply by mycx001** [reply]
Thanks for your reply. What I want to say is that of course I use my own API for my own store, and I don't provide the key to others. I just ask if there is a simple standard coding program similar to a template or an officially recognized method that I can use as a reference to make a program for my own personal use, so as to improve the efficiency of processing order information so that I can have more time to design and process better products, that's all.

**Comment by turboturtleD** [comment]
Etsy has a pretty good quick start tutorial here: https://developers.etsy.com/documentation/tutorials/quickstart/

**Reply by mycx001** [reply]
Thank you for your response. I've been looking at this tutorial, but I haven't found the part I need. Could you please tell me which specific part it is? I need to export new orders, including notes and gift messages.
